sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -14,6 +14,7 @@ export interface InstitutionLocationTypeDto {
   locationName: string;
   postalZipCode: string;
   provinceState: string;
+  institutionCode: string;
```

Comment:
This DTO is used to create/update(POST/PATCH) the institution.
During the POST I can see a dry run that is running the validations, but this does not happen on update/PATCH.
Can you please add the dry/run on the update also?
The reason that I am asking is this PR is because the new field, during the update, is not receiving any server validation and it would fail only on the database if it has more than 4 characters, what means that I can still save invalid data.

Reply:
DB wont allow more than 4 characters, but it will allow less than 4 character and lower case and all other characters, yes, it make sens to add dry/run in patch. updating